pkg/record/schema/proto/mutable_test.go

```diff
@@ -106,23 +114,27 @@ func TestMutableSchema_SetDescriptors_Panics(t *testing.T) {
 
 			ms := fileDescriptorSetToMutalbeSchema(t, getFileDescriptorSet(t, standaloneDescriptorSetPath))
 			ms.SetDescriptors(tc.descriptors)
-			assert.True(t, false, "expected panic")
+			is.True(false)
```

Comment:
It'd be good to keep the message `expected panic`, because otherwise `is.True(false)` on its own would be confusing.